modules/common/src/main/scala/surge/internal/streams/ReplayCoordinator.scala

```diff
@@ -71,6 +72,9 @@ class ReplayCoordinator(topicName: String, consumerGroup: String, registry: Acto
     self ! ResumeConsumers
   }
 
+  override def getReplayProgress: Future[ReplayProgress] =
+    self.ask(GetReplayProgress)(TimeoutConfig.ReplayCoordinatorActor.actorAskTimeout).mapTo[ReplayProgress]
```

Comment:
Must have missed this in the other PR but the `self.ask` here is a bit odd. In general any communication to the actor should be through messages rather than direct method calls on it. If we want to separate out an API that's easy to consume from other places but back the logic by the actor we can follow a similar pattern to the `KafkaStreamManager` stuff.

Reply:
done